pkg/kapis/iam/v1alpha2/handler.go

```diff
@@ -980,6 +997,14 @@ func (h *iamHandler) CreateClusterMembers(request *restful.Request, response *re
 		}
 	}
 
+	if event := requestinfo.AuditEventFrom(request.Request.Context()); event != nil {
```

Comment:
Same here, I think we'd better put auditing aside from these modules, to do better decoupling.